### DISCUSSION TITLE
[Feature] ML model preloading

### DISCUSSION THREAD

**Discussion by bo0tzz** [immich-app/immich#7411] [Feature Request]
### The feature

Add an `IMMICH_MODEL_PRELOAD` environment variable to the machine-learning container. Any models listed (comma-separated) in this env var are loaded immediately on startup of the container, and kept in memory (ie not unloaded after the ttl expiry). This is primarily useful for search response times on the CLIP text model.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by jrasm91** [comment]
Implemented in #7540
